tests/UI/specs/Menus_spec.js

```diff
@@ -105,6 +105,26 @@ describe("Menus", function () {
         expect(await page.screenshot({ fullPage: true })).to.matchImage('mobile_top');
     });
 
+    it('should load the TagManager menu correctly on mobile', async function() {
+        page.webpage.setViewport({ width: 768, height: 512 });
```

Comment:
Wouldn't it make more sense to have that test as part of the TagManager. If we have that in core, we need to update the results here when you do a change in tagmanager to the menu...

Reply:
yaa even that makes sense, will remove the tests and added in plugin

Reply:
@sgiehl Tests removed, does the PR looks good to you ?